scripts/gen_help_html.lua

```diff
@@ -2,32 +2,38 @@
 --
 -- NOTE: :helptags checks for duplicate tags, whereas this script checks _links_ (to tags).
 --
+-- USAGE (For CI/local testing purposes): Simply `make lintdoc` or `scripts/lintdoc.lua`, which
+-- basically does the following:
+--   1. :helptags ALL
+--   2. nvim -V1 -es +"lua require('scripts.gen_help_html').run_validate()" +q
+--   3. nvim -V1 -es +"lua require('scripts.gen_help_html').test_gen()" +q
+--
 -- USAGE (GENERATE HTML):
---   1. Run `make helptags` first; this script depends on vim.fn.taglist().
---   2. nvim -V1 -es --clean +"lua require('scripts.gen_help_html').gen('./build/runtime/doc/', 'target/dir/')"
+--   1. `:helptags ALL` first; this script depends on vim.fn.taglist().
+--   2. nvim -V1 -es --clean +"lua require('scripts.gen_help_html').gen('$VIMRUNTIME/doc', 'target/dir/')" +q
```

Comment:
`+q` shouldn't be needed with `-es`

Reply:
The reason why I put this is to remove unnecessary message when `-V1` is used:

```diff
 ❯❯❯ nvim -V1 -es -c 'lua error()'
 Error detected while processing command line:
 E5108: Error executing lua [NULL]
-Entering Ex mode.  Type "visual" to go to Normal mode.
-:%
```
(note: it preserves the exit code well depending on whether error `:message` was emitted) So let's keep it?

Reply:
Need to fix `-es` so it doesn't print that message :/